### DISCUSSION TITLE
Possibly a big discovery

### DISCUSSION THREAD

**Discussion by ForYouDevleopment** [Noobgamer0111/SubwaySurfersDecrypted#4] [Ideas]
Ok so I was searching through files just trying to find a way to get some discovery
and I randomly stumbled upon
tower/gamedata/manifest
Then open the file.
Go to the objects part and choose one of the objects. There's the file name and a KEY.
Maybe we can do something with that.

There has always been "key" there to. Even before encrypted data.

**Comment by Noobgamer0111** [comment]
I've already tried that in my initial analysis of the installed app. It doesn't seem to correspond to anything IIRC.

But if you are able to figure how to get the app into a debuggable state and use an emulator to grab memory dumps, we can then grab the cipher to decrypt it.

**Comment by Matsn-8Bit** [comment]
So I just did a bit of a research that you can replace the new enceypted json file with the old one LUCKILY i have the old json file its this thing 
```html
{ "version": 2, "data": "{\"lastSaved\":\"\",\"patchVersion\":0,\"currencies\":{\"1\":{\"value\":0,\"expirationType\":0},\"2\":{\"value\":0,\"expirationType\":0},\"3\":{\"value\":2100000000,\"expirationType\":0},\"4\":{\"value\":2100000000,\"expirationType\":0},\"5\":{\"value\":2100000000,\"expirationType\":0},\"6\":{\"value\":2100000000,\"expirationValue\":0,\"expirationType\":1}},\"20\":{\"value\":0,\"expirationType\":0},\"21\":{\"value\":0,\"expirationType\":0},\"22\":{\"value\":0,\"expirationType\":0},\"23\":{\"value\":0,\"expirationType\":0},\"24\":{\"value\":0,\"expirationType\":0},\"lootboxQueue\":{\"unopenedLootboxes\":{\"0\":[],\"2\":[]}},\"currencyAllowedInRun\":{\"5\":true,\"4\":true},\"lootBoxesOpened\":{\"mini_mystery_box\":3,\"mystery_box\":7,\"token_box\":16,\"super_mystery_box\":5},\"ownedOnlyBuyOnceProducts\":[\"free_token_box\",\"multiplier_pack\",\"speed_crew\"]}"}
```

**Reply by Matsn-8Bit** [reply]
Im going to research how to change the cities etc hope this help <3 Also yes when u try to boot the subway surfer and wanted to change the json file (old file) it resseted to the new json file im still going to try to find the key to make it not resseted
